tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -3344,8 +3369,97 @@ TEST (test_tensor_filter, reload_tflite_same_model_wrong_dims_n)
   g_free (test_model_renamed);
 }
 
+/**
+ * @brief Test framework auto detecion option in tensor-filter.
+ */
+TEST (test_tensor_filter, framework_auto_ext_tflite_01)
+{
+  GET_MODEL_PATH(mobilenet_v1_1.0_224_quant.tflite)
```

Comment:
Q: there are a lot of cases using without blank `XXX(` and with blank`YYY (` in this project. Is there any standard for macro?

Reply:
Execution line can have a blank space. It was my mistake. I will fix it.